wpimath/src/main/java/edu/wpi/first/math/util/Units.java

```diff
@@ -176,4 +176,45 @@ public static double kilogramsToLbs(double kilograms) {
   public static double lbsToKilograms(double lbs) {
     return lbs * kKilogramsPerLb;
   }
+
+  /**
+   * Converts falcon counts into degrees.
+   *
+   * @param counts The number of counts returned by the falcon.
+   * @return Degrees converted from falcon counts.
+   */
+  public static double falconCountsToDegrees(double counts) {
+    return counts * 360 / 2048;
+  }
+
+  /**
+   * Converts degrees into falcon counts.
+   *
+   * @param degrees The degrees to convert to falcon counts.
+   * @return Falcon counts converted from degrees.
+   */
+  public static double degreesToFalconCounts(double degrees) {
+    return degrees * 2048 / 360;
+  }
+
+  /**
+   * Converts falcon counts into RPM.
+   *
+   * @param counts The velocity returned by the falcon. (Sensor units per 100 milliseconds)
+   * @return RPMs converted from falcon velocity.
+   */
+  public static double falconVelocityToRPM(double counts) {
+    return counts * 600 / 2048;
```

Comment:
Can we document somewhere where these values come from?

Reply:
Where should I add the unit tests to? Also, I can try out the C++ implementation!

Reply:
I can add the unit tests, but could someone help out with the c++?

Reply:
Yes, I can bring it up in our weekly standup and we'll comment here if there are any updates.